src/js/components/VoteTable/VoteTableHeader.vue

```diff
@@ -41,6 +41,17 @@
 			</div>
 		</div>
 
+		<div v-if="acl.allowEdit && poll.type === 'datePoll'" class="new-event">
+			<a class="button"
+				:class="{
+					primary: isWinner,
+					'icon-calendar': isWinner,
+					'icon-calendar-000': !isWinner
```

Comment:
You could put this into the component as template, add some props  and make it reusable.